src/Polly.Core/CircuitBreaker/BrokenCircuitException.TResult.cs

```diff
@@ -0,0 +1,46 @@
+#if !NETCOREAPP
+using System.Runtime.Serialization;
+#endif
+
+namespace Polly.CircuitBreaker;
+
+#pragma warning disable CA1032 // Implement standard exception constructors
```

Comment:
Maybe in a later PR we add a few more of the expected constructors (at least ones that make sense)?

Reply:
Here is tricky, because these do require `TResult` as input. Is it a big problem if we don't have these?

Reply:
I was just wondering if we should add, say, `.ctor(string, Exception, TResult)` (maybe swap the order of parameters) - I would agree that `.ctor()` probably doesn't make any sense.